pillar/heroku/discussions.sls

```diff
@@ -0,0 +1,177 @@
+      'OPEN_DISCUSSIONS_BASE_URL': 'https://open.mit.edu',
+      'OPEN_DISCUSSIONS_COOKIE_DOMAIN': 'mit.edu',
+      'OPEN_DISCUSSIONS_SUPPORT_EMAIL': 'odl-discussions-support@mit.edu',
+      'release_branch': 'release',
+      'SOCIAL_AUTH_MICROMASTERS_LOGIN_URL': 'https://micromasters.mit.edu/login/edxorg/?next=/discussions/',
+      'vault_env_path': 'production-apps'
+      }
+} %}
+{% set env_data = env_dict[environment] %}
+{% set business_unit = 'mit-open' %}
+{% set pg_creds = salt.vault.cached_read('postgres-{}-apps-opendiscussions/creds/opendiscussions'.format(env_data.env_name), cache_prefix='heroku-opendiscussions') %}
+{% set rds_endpoint = salt.boto_rds.get_endpoint('{env}-rds-postgres-opendiscussions'.format(env=env_data.env_name)) %}
+
+proxy:
+  proxytype: heroku
+
+heroku:
+  app_name: odl-open-discussions-{{ env_data.env_name }}
+  api_key: __vault__::secret-operations/global/heroku/api_key>data>value
+  config_vars:
+    ALGOLIA_API_KEY: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/algolia>data>api_key
+    ALGOLIA_APP_ID: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/algolia>data>app_id
+    ALLOWED_HOSTS: '[*]'
+    AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/read-write-delete-xpro-app-{{ env_data.env_name }}>data>access_key
+    AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/read-write-delete-xpro-app-{{ env_data.env_name }}>data>secret_key
+    AWS_STORAGE_BUCKET_NAME: 'odl-discussions-{{ env_data.env_name }}'
+    CELERYD_CONCURRENCY: # ONLY IN RC
+    CKEDITOR_ENVIRONMENT_ID:  __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/ckeditor>data>environment_id
+    CKEDITOR_SECRET_KEY:  __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/ckeditor>data>secret_key
+    CKEDITOR_UPLOAD_URL:  __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/ckeditor>data>upload_url
+    CLOUDFRONT_DIST: {{ env_data.CLOUDFRONT_DIST }}
+    {% if env_data.env_name != 'ci' %}
+    DATABASE_URL: postgres://{{ pg_creds.data.username }}:{{ pg_creds.data.password }}@{{ rds_endpoint }}/opendiscussions
+    {% endif %}
+    DEBUG: {{ env_data.DEBUG }}
+    EDX_API_CLIENT_ID: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/edx-api-client>data>id
+    EDX_API_CLIENT_SECRET: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/edx-api-client>data>secret
+    EDX_API_URL: https://api.edx.org/catalog/v1/catalogs/10/courses
+    ELASTICSEARCH_HTTP_AUTH: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/elasticsearch>data>http_auth
+    ELASTICSEARCH_INDEX: {{ env_data.ELASTICSEARCH_INDEX}}
+    ELASTICSEARCH_URL: {{ env_data.ELASTICSEARCH_URL}}
+    EMBEDLY_KEY: __vault__::secret-operations/{{ env_data.vault_env_path }}/{{ business_unit }}/embedly_key>data>value
+    ENABLE_STUNNEL_AMAZON_RDS_FIX: True
```

Comment:
This setting doesn't actually do anything anymore with the updated pgbouncer buildpack